homeassistant/components/automation/__init__.py

```diff
@@ -254,6 +255,19 @@ def state_attributes(self):
             ATTR_LAST_TRIGGERED: self._last_triggered
         }
 
+    @asyncio.coroutine
+    def async_added_to_hass(self) -> bool:
+        """Call when you add."""
+        state = yield from async_get_last_state(self.hass, self.entity_id)
+        if state is None:
+            return
+        res = state.state == STATE_ON
+        if res != self._enabled:
+            if self._enabled:
+                yield from self.async_turn_off()
+            else:
+                yield from self.async_turn_on()
+
```

Comment:
Not very important, but it would be cool to pull in `last_triggered` here as well.